fastlane/README.md

```diff
@@ -38,6 +38,6 @@ Ship to Playstore Beta.
 
 ----
 
-This README.md is auto-generated and will be re-generated every time [fastlane](https://fastlane.tools) is run.
+This README.md is auto-generated and will be re-generated every time [_fastlane_](https://fastlane.tools) is run.
```

Comment:
these changes are autogenerated, so the file is updated every time Fastlane runs.. so better commit it and forget about it 😅